### PR TITLE
Adblocking response AB test - part 1

### PR DESCRIPTION
**Pull request by Calanthe**
## What does this change?

This PR introduces the first part of an AB test which will test how users will response to adblock messages. 
## What is the value of this and can you measure success?

The message (which is not decided yet, there will be 2 different variants) will be shown to adblock users on content pages to encourage them either to whitelist our site or become a paid member. The message box will be on a dark background layer which will cover the whole content and will not be dismissible.
## Does this affect other platforms - Amp, Apps, etc?

no
## Screenshots

screen shot 2016-03-15 at 16 44 36
## TODO in the next PRs:
- A proper links, text, cta's, audience and data links.
- `Enter subscriber number` page which based on the provided number will check if the user is a subscriber
- Logic with redirect to the last seen article to show it after whitelisting, entering subscriber's number or logging as a member. 
## Request for comment

@uplne @jbreckmckye 
## 

_Does this PR meet the [contributing guidelines](https://github.com/guardian/frontend/blob/issue_pr_templates/.github/CONTRIBUTING.md#submission)?_

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/experiments/tests/adblocking-response.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        };
+
+        this.variants = [{
+            id: 'control',
+            test: function () {}
+        }, {
+            id: 'variantA',
+            test: function () {
+                //TODO check also if not a subscriber
+                if (detect.adblockInUseSync() && !config.page.isFront && !userFeatures.isPayingMember()) {
```
Comment (jbreckmckye) [inline static/src/javascripts/projects/common/modules/experiments/tests/adblocking-response.js]: The synchronous API is going to be deprecated soon - as soon as the omniture top-up call is removed.

File: static/src/javascripts/projects/common/modules/experiments/tests/adblocking-response.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    contains,
+    userFeatures,
+    SurveySimple
+) {
+    return function () {
+        this.id = 'AdblockingResponse';
+        this.start = '2016-03-21';
+        this.expiry = '2016-03-31';
+        this.author = 'Zofia Korcz';
+        this.description = 'Adblocking respoonse test';
```
Comment (uplne) [inline static/src/javascripts/projects/common/modules/experiments/tests/adblocking-response.js]: We can call the new framework 'respoonse.js'

**Comment by paperboyo** [comment]
:sob::gun:

10vq38

> — It was me.

> — Excuse me?

> — If it wasn’t you,
it must have been me.

> — Not at all. Please.

_from https://www.youtube.com/watch?v=IkEf42l6vo4_

**Comment by jbreckmckye** [comment]
:+1: once the conflict's resolved

**Pull request merged**